oddbird/sass-site
# [blog-width] Limit the width of blog column

![](https://source.unsplash.com/featured/?cute,animal&fjkhjbn )


## Description
Use the smaller container for blog posts to increase readability. 

## Steps to test/reproduce
view a blog single like /blog/sass-and-native-nesting/ and check the width of blog content is restricted.


## Show me
![Screenshot 2023-05-18 at 10 01 11 PM](https://github.com/oddbird/sass-site/assets/1581694/54d0a218-4464-4e6d-83d4-d8f7e52af656)
